### ISSUE TITLE
Syntax Error in URL 

### ISSUE THREAD

**Issue by bartman0815** [petercerno/good-morning#7]
When I use your simple 3 line code example:

import good_morning as gm
kr = gm.KeyRatiosDownloader()
kr_frames = kr.download('AAPL')

following failure appears:

$ python3 test.py 
Traceback (most recent call last):
  File "test.py", line 1, in <module>
    import good_morning as gm
  File "/home/pi/good-morning-master/morningstar/good_morning.py", line 59
    url = (ur'http://financials.morningstar.com/ajax/exportKR2CSV.html?' +
                                                                       ^
SyntaxError: invalid syntax

**Comment by dr333** [comment]
Encountered the same issue.

**Comment by linwoodc3** [comment]
I think I figured out the fix @petercerno.  Somehow, during the last push I sent, the urls beginning with `r'https.........` got replaced with `ur'https:.........`.  Or, in simpler terms, a `u` got put on each string.  

And then another problem comes with urllib2 (I don't remember doing this and didn't see it in my commits..but it's there).  

I found a way to fix both of them (just return text back to Peter's Nov 2015 commit).  I'll make the changes and do the pull request

This new pull request will also close #6
